src/redux/reducer/todoReducer.js

```diff
@@ -0,0 +1,44 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+const todoReducer = createSlice({
+  name: "todos",
+  initialState: { todos: [] },
+  reducers: {
+    addTodo: (state, action) => {
+      state.todos.push(action.payload);
+    },
+
+    updateTodo: (state, action) => {
+      state.todos = state.todos.map((todo) => {
```

Comment:
this is the wrong way to do this, because if you have 100 records, then the code will map the whole 100 records.
what if the user updated 1st records? isn't it bad to iterate it 99 times more? @amitark7